streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingJobProgressListener.scala

```diff
@@ -40,6 +45,8 @@ private[streaming] class StreamingJobProgressListener(ssc: StreamingContext)
   private var totalProcessedRecords = 0L
   private val receiverInfos = new HashMap[Int, ReceiverInfo]
 
+  private val batchTimeToJobIds = new HashMap[Time, ArrayBuffer[(OutputOpId, JobId)]]
```

Comment:
`job` in `batchTimeToJobIds ` is confusing as there is a streaming Job, and a spark job. Please call it `sparkJob` wherever you are referring to spark jobs. In fact, call the type `JobId` as `SparkJobId`.